my-dsl.md

```diff
@@ -2,37 +2,51 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+[SQLAlchemy](http://www.sqlalchemy.org)
 
 # Domain
 _Describe the language's domain in five words._
 
+SQL databases during python programming.
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
 
+The SQLAlchemy code is translated into true SQL and executed as such. The model depends slightly on which SQL database being used but can translated into code to model, insert, select, join etc.
+
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+SQLAlchemy is closer almost purely domain specific. It can only be used for this one domain and is not turing complete.
```

Comment:
It seems like you could also think of SQLAlchemy as a library instead of its own DSL. Can you think of any arguments why it might be one of the other?